cmd/provider/main.go

```diff
@@ -94,12 +99,23 @@ func main() {
 	kingpin.FatalIfError(err, "Cannot create controller manager")
 	kingpin.FatalIfError(apis.AddToScheme(mgr.GetScheme()), "Cannot add Template APIs to scheme")
 
+	metricRecorder := managed.NewMRMetricRecorder()
```

Comment:
for these metrics additions, did you use https://github.com/crossplane-contrib/provider-kubernetes/pull/224/files as a model to follow?

Reply:
Yes, I think I took some references from the PR to add metrics previously. So should follow the same logic.